lib/puppet/module.rb

```diff
@@ -56,7 +56,7 @@ def self.is_module_namespaced_name?(name)
   attr_writer :environment
 
   attr_accessor :dependencies, :forge_name
-  attr_accessor :source, :author, :version, :license, :puppetversion, :summary, :description, :project_page
+  attr_accessor :source, :author, :version, :license, :summary, :description, :project_page
```

Comment:
Will add the explicit getter/setter. 

From what I understand, this was a never-documented metadata key (aside from the source, of course), and affects 6 modules on the Forge with values ranging from 0.25 to 2.6 to 2.6.4. Since the validation is a string match, I think it would only allow one of those to actually work. So, I'd say just a release note about deprecation should be fine.